teuthology/orchestra/remote.py

```diff
@@ -150,6 +155,10 @@ def machine_type(self):
             self._machine_type = remote_info.get("machine_type", None)
         return self._machine_type
 
+    @property
+    def is_reimageable(self):
```

Comment:
I wonder why do we need `is_` prefix?

Reply:
because that's fairly conventional?

````
$ git grep def\ is_ 
teuthology/lock/query.py:def is_vm(name=None, status=None):
teuthology/ls.py:    def is_job_dir(parent, subdir):
teuthology/misc.py:def is_type(type_, cluster=None):
teuthology/misc.py:def is_in_dict(searchkey, searchval, d):
teuthology/orchestra/remote.py:    def is_reimageable(self):
teuthology/orchestra/remote.py:    def is_online(self):
teuthology/orchestra/remote.py:    def is_vm(self):
teuthology/provision/pelagos.py:    def is_task_active(self, task_url):
teuthology/prune.py:def is_old_enough(file_name, days):
teuthology/repo_utils.py:def is_fresh(path):
teuthology/task/hadoop.py:def is_hadoop_type(type_):
````

Reply:
ok, thanks